api/docs/open_api_schema.json

```diff
@@ -654,23 +654,6 @@
         },
         "description": "Known actuals data."
       },
-      "CovidPatientsMethod": {
```

Comment:
I'm not sure why these entered the diff. I have the latest master checked out, but running schema generation includes these changes. Please advise?

Reply:
I think you may need to update your requirements.  We bumped the version of pydantic which changed the schema generation a bit.